easybuild/tools/module_generator.py

```diff
@@ -174,25 +174,6 @@ def get_modules_path(self, fake=False, mod_path_suffix=None):
 
         return os.path.join(mod_path, mod_path_suffix)
 
-    def prepare(self, fake=False):
```

Comment:
Yes, it's not used anywhere else, and easyblocks shouldn't be calling `self.module_generator.prepare` at all (none of the ones in the central repo do, I checked), I don't see the point of that...